### PR TITLE
OSDOCS-2627: Adding docs for admin ack required to upgrade to OCP 4.9

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-2627

Previews: 
* OCP: https://deploy-preview-36809--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-prepare.html
* OKD: http://file.rdu.redhat.com/~ahoffer/2021/OSDOCS-2627-49-upgrading/OSDOCS-2627-49-upgrading/updating/updating-cluster-prepare.html

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@wangke19 / @jianlinliu Can you please QE review this update? This is almost the same content as the kbase article, with just some minor wording and formatting changes. All that needs to be done is replacing the "TODO" with the actual 4.8.z release that it goes out in.

Previews: 
* OCP: https://deploy-preview-36809--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-prepare.html
* OKD (requires VPN): http://file.rdu.redhat.com/~ahoffer/2021/OSDOCS-2627-49-upgrading/OSDOCS-2627-49-upgrading/updating/updating-cluster-prepare.html

File: modules/update-preparing-evaluate-alerts.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-prepare.adoc
+
+[id="update-preparing-evaluate-alerts_{context}"]
+= Reviewing alerts to identify uses of removed APIs
+
+{product-title} 4.8 introduced two new alerts that fire when an API that will be removed in the next release is in use:
```
Comment (bobfuru) [inline modules/update-preparing-evaluate-alerts.adoc:8]: I find this sentence accurate but a bit hard to follow. Perhaps it could be more direct as something like, "...introduced two new alerts that tell you when you are using an API that will be removed in the next release" or even a small transposition, such as: "...when an API is in use that will be removed in the next release". Up to you if you find either of these helpful or not. The more I read it, though, maybe your original form does make the most sense. 🤷

File: modules/update-preparing-evaluate-alerts.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+* `APIRemovedInNextReleaseInUse` - for APIs that will be removed in the next {product-title} release.
+* `APIRemovedInNextEUSReleaseInUse` - for APIs that will be removed in the next {product-title} Extended Update Support (EUS) release.
+
+If either of these alerts are firing in your cluster, review the alerts and take action to clear the alerts by migrating manifests and API clients to use the new API version. You can use the `APIRequestCount` API to get more information about the APIs in use and which workloads are using removed APIs.
```
Comment (bobfuru) [inline modules/update-preparing-evaluate-alerts.adoc:13]:
```suggestion
If either of these alerts are firing in your cluster, review the alerts and take action to clear the alerts by migrating manifests and API clients to use the new API version. You can use the `APIRequestCount` API to get more information about which APIs are in use and which workloads are using removed APIs.
```
for parallelism?

File: modules/update-preparing-evaluate-apirequestcount-workloads.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+You can examine the `APIRequestCount` resource for a given API version to help identify which workloads are using the API.
+
+.Prerequisites
+
+* You must have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+* Run the following command and examine the `username` and `userAgent` fields to help identify the workloads using the API:
```
Comment (bobfuru) [inline modules/update-preparing-evaluate-apirequestcount-workloads.adoc:16]: For clarity, I'd suggest "the workloads that are using the API" here

File: modules/update-preparing-evaluate-apirequestcount.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-prepare.adoc
+
+[id="update-preparing-evaluate-apirequestcount_{context}"]
+= Using APIRequestCount to identify uses of removed APIs
+
+You can use the `APIRequestCount` API to track API requests and review whether any are using one of the removed APIs.
```
Comment (bobfuru) [inline modules/update-preparing-evaluate-apirequestcount.adoc:8]: "any are" threw me at first read, maybe "any of them are" for readability?

File: modules/update-preparing-list.adoc [hunk 5]
```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-prepare.adoc
+
+[id="update-preparing-list_{context}"]
+= Removed Kubernetes APIs
+
+// TODO: Keep michael's section in the release notes (which this duplicates), or link to this from his RN section?
+{product-title} 4.9 uses Kubernetes 1.22, which removed the following deprecated `v1beta1` APIs. Migrate manifests and API clients to use the `v1` API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22[Kubernetes documentation].
```
Comment (bobfuru) [inline modules/update-preparing-list.adoc:9]: Consider starting the second sentence as, "You must migrate manifests..." or even "You must migrate API clients and manifests..." to avoid the alliteration. 😁

File: updating/updating-cluster-prepare.adoc [hunk 6]
```diff
@@ -0,0 +1,37 @@
+[id="updating-cluster-prepare"]
+= Preparing to update to {product-title} 4.9
+include::modules/common-attributes.adoc[]
+:context: updating-cluster-prepare
+
+toc::[]
+
+{product-title} 4.9 uses Kubernetes 1.22, which removed a significant number of deprecated `v1beta1` APIs.
+
+4.8.TODO introduced a requirement that an administrator provide a manual acknowledgment before the cluster can be upgraded from {product-title} 4.8 to 4.9. This is to help prevent issues after upgrading to {product-title} 4.9, where APIs that have been removed are still in use by workloads, tools, or other components running on or interacting with the cluster. Administrators must evaluate their cluster for any APIs in use that will be removed and migrate the affected components to use the appropriate new API version. After this is done, the administrator can provide the administrator acknowledgment.
```
Comment (bobfuru) [inline updating/updating-cluster-prepare.adoc:10]: Suggest "that an administrator must provide" to avoid ambiguity.

**Review by bobfuru — COMMENTED** [review]
This all looks really good! I left several wording suggestions for consideration but they're pretty minor. Otherwise, LGTM.

**Comment by bergerhoffer** [comment]
Thanks @bobfuru for the great suggestions.

Merging since we have QE approval (and the wording adjustments don't change any technical meanings).
